elastic/kibana
# [SecuritySolution] Fix legend content for events histogram

## Summary

https://github.com/elastic/kibana/issues/89205
Steps to verify:
1. Go to network / Flows
2. Hover on any ip in Source IPs or Destination IPs table and select `show top source.ip` or `show top destination.ip`

exceptions:
If query success, it should display ips correctly in the legend.
If query failed, it should close the histogram panel and show an error toast.

### Checklist
